tests/test_limits.py

```diff
@@ -1,19 +1,26 @@
 import os
 
 import pytest
-from qiskit import IBMQ
-from qiskit.providers.aer import AerSimulator
+from qiskit_ibm_provider import IBMProvider
+from qiskit_aer import AerSimulator
 from qiskit_braket_provider import AWSBraketProvider, BraketLocalBackend
 
 from qbench.limits import get_limits
 from qbench.testing import MockSimulator
 
+# TODO IBMQ_TOKEN is deprecated by now
 IBMQ_TOKEN = os.getenv("IBMQ_TOKEN")
+QISKIT_IBM_TOKEN = os.getenv('QISKIT_IBM_TOKEN')
 
 
 @pytest.fixture(scope="module")
 def ibmq_provider():
-    return IBMQ.get_provider() if IBMQ.active_account() else IBMQ.enable_account(IBMQ_TOKEN)
+    # TODO Maybe stop supporting IBMQ_TOKEN variable?
+    if sum(e in os.environ for e in ('QISKIT_IBM_TOKEN', 'IBMQ_TOKEN')) > 0:
```

Comment:
Also, the documentation now suggests another name: `IQP_API_TOKEN` https://quantum.cloud.ibm.com/docs/en/guides/cloud-setup

Reply:
Commit 790d2b4